### ISSUE TITLE
Some failures caused by PartonLevel module

### ISSUE THREAD

**Issue by marco-link** [WbWbX/nanoAOD-tools#9]
When processing the `WbjToLNu` dataset, some jobs fail with:
```
Traceback (most recent call last):
  File "processors/ST.py", line 435, in <module>
    p.run()
  File "/work/mlink/wb_scattering/CMSSW_11_1_7/python/PhysicsTools/NanoAODTools/postprocessing/framework/postprocessor.py", line 234, in run
    eventRange=eventRange, maxEvents=self.maxEntries
  File "/work/mlink/wb_scattering/CMSSW_11_1_7/python/PhysicsTools/NanoAODTools/postprocessing/framework/eventloop.py", line 78, in eventLoop
    ret = m.analyze(e)
  File "/work/mlink/wb_scattering/CMSSW_11_1_7/python/PhysicsTools/NanoAODTools/modules/PartonLevel.py", line 89, in analyze
    if (genParticles[motherIdx].pdgId==21):
  File "/work/mlink/wb_scattering/CMSSW_11_1_7/python/PhysicsTools/NanoAODTools/postprocessing/framework/datamodel.py", line 71, in __getattr__
    val = val[self._index]
ValueError: none of the 2 overloaded methods succeeded. Full details:
  int& TTreeReaderArray<int>::operator[](unsigned long idx) =>
    could not convert argument 1 (can't convert negative value to unsigned long)
  const int& TTreeReaderArray<int>::operator[](unsigned long idx) =>
    could not convert argument 1 (can't convert negative value to unsigned long)
```
To reproduce run:
```
python2 processors/ST.py --year 2017 --isSignal --input=root://cmsxrootd-redirectors.gridka.de//store/user/mlink/WbChargeReco/2022-03-01_v6/mc_2017/WbjToLNu_4f_TuneCP5_13TeV-madgraph-pythia8/WbChargeReco_mc_2017_2022-03-01_v6/220301_100349/0000/nanox_108.root
```

**Comment by marco-link** [comment]
please assign me

**Comment by marco-link** [comment]
fixed by including the most recent Parton level module (included in #8)
